docs/markdown/Getting Started/getting-started/prerequisites.md

```diff
@@ -68,3 +68,9 @@ System-specific notes
 > - Install a recent Linux distribution under WSL 2 (we have tested with Ubuntu 20.04 LTS).
 > - Run `sudo apt install unzip python3-dev python3-distutils python3-venv gcc` in the distribution.
 > - You can then run Pants commands in a Linux shell, or in a Windows shell by prefixing with `wsl `.
+
+> 🚧 Using WSL on Windows
```

Comment:
Since we already have a callout for Windows, I think it's sufficient if we add this as another bullet point above?